doc/Requirements Specification/Parts/Functional Requirements.lyx

```diff
@@ -838,14 +806,21 @@ gls{PCM}
 
 \end_inset
 
- instance provided to Beagle will be found in the 
-\begin_inset CommandInset ref
-LatexCommand ref
-reference "result PCM"
+.
+ The modified 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{PCM}
+\end_layout
 
 \end_inset
 
-.
+ will only be extended.
```

Comment:
Why the complication? I think the initial wording was very precise and conveyed exactly what we plan to do. What is your problem with it?